spec/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'My post', text: 'My post body', author_id: 1, commentsCounter: 12, likesCounter: 3) }
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
```

Comment:
- [ ] As mentioned earlier, to make sure any custom methods in the models should have a corresponding unit test, Please add a test that will assert the `comments_for_post` and `update_user_posts_counter` methods, which are inside the `post` model.

Reply:
Fixed.